presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -772,6 +772,46 @@ Examples:
 
     CALL iceberg.system.expire_snapshots(schema => 'schema_name', table_name => 'table_name', snapshot_ids => ARRAY[10001, 10002]);
 
+Rewrite Data Files
+^^^^^^^^^^^^^^^^^^
+
+In Iceberg, we tracks all data files under different partition specs in a table. More data files leads to
+more metadata stored in manifest files, and small data files causes an unnecessary amount of metadata and
+less efficient queries from file open costs. Besides, data files under different partition specs might
+prevent metadata level deletion or predicate thoroughly push down for Presto engine.
+
+Procedure `rewrite_data_files` is used for rewriting the data files of the specified table so that they are
+merged into fewer but larger files under the newest partition spec. If the table is partitioned, the data
+files compaction could act separately on the selected partitions. This can improve read performance by reducing
+metadata overhead and runtime file open cost.
+
+The following arguments are available:
+
+===================== ========== =============== =======================================================================
+Argument Name         required   type            Description
+===================== ========== =============== =======================================================================
+``schema``            ✔️         string          Schema of the table to update
+
+``table_name``        ✔️         string          Name of the table to update
+
+``filter``                       string          predicate as a string used for filtering the files. Note that currently
```

Comment:
```suggestion
``filter``                       string          Predicate as a string used for filtering the files. Currently
```
When you address these suggestions, please put the line breaks in the correct places for this entry for the table format. (To be clear: your draft formatting is great! I'm asking that you check to make sure my suggestions don't introduce a format error for the table.)

Reply:
Thanks for the reminder, got it.